utils/python/CIME/utils.py

```diff
@@ -799,3 +797,20 @@ def transform_vars(text, case=None, subgroup=None, check_members=None, default=N
 def get_my_queued_jobs():
     # TODO
     return []
+
+def wait_for_unlocked(filepath):
```

Comment:
What is this for? I see that you've added this function, but I don't see it being called anywhere?